android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/FormConfigurationJsonFormActivity.java

```diff
@@ -38,11 +39,22 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
-        formUtils = new FormUtils();
+        formUtils = getFormUtils();
         JSONObject jsonObject = getmJSONObject();
         checkIfFormUpdate(jsonObject);
     }
 
+    private FormUtils getFormUtils(){
+        if (!this.isUsingLocaleForms()){
```

Comment:
I think this should have been flagged by codacy, there should be a space after the last bracket :smile: . Something small.